ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -2321,3 +2324,159 @@ export const testWidth = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testScrollInsideBox = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      await whisper.create({
+        label: 'Scrolling Inside Box Test',
+        onClose: () => {
+          console.debug('closed');
+        },
+        components: [
+          {
+            body: 'Does is scroll?',
+            type: WhisperComponentType.Markdown,
+          },
+
+          {
+            type: whisper.WhisperComponentType.Box,
+            direction: Direction.Horizontal,
+            justifyContent: JustifyContent.SpaceEvenly,
+            // eslint-disable-next-line no-underscore-dangle
+            customHeight: CustomHeight.Small,
+            children: [
+              {
+                type: WhisperComponentType.TextInput,
+                label: 'TextInput',
+                onChange: (value) => {
+                  console.debug(`Input value changed: ${value}`);
+                },
+              },
+              {
+                type: whisper.WhisperComponentType.Markdown,
+                body: stripIndent`
+              A paragraph with *emphasis* and **strong importance**.
+              this is the test of scroll feature
+              
+              
+              
+            
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut molestie feugiat est, eget mollis metus sollicitudin tincidunt. Integer ut lorem in nisl gravida placerat sed eget eros. Fusce finibus velit quam, in tincidunt dui consectetur rutrum. Donec venenatis sem in nulla malesuada, a lobortis nisi volutpat. Vivamus vitae posuere quam. Praesent eget vestibulum risus. Fusce posuere diam elit, eu aliquet nunc laoreet nec. Vestibulum viverra turpis ac ligula euismod, eget mollis ligula faucibus. Nam dictum viverra sem in imperdiet. Donec tempus ullamcorper rhoncus.
```

Comment:
Could I suggest to move this text to a constant in another file? whisper/index.ts is growing large and we should start thinking on how to split it and make it smaller for a better maintenance.